src/main/java/org/broadinstitute/consent/http/models/Dac.java

```diff
@@ -32,6 +32,8 @@ public class Dac {
 
   private String email;
 
+  private DataAccessAgreement associatedDaa;
```

Comment:
We should add this new field to the api docs as well: `src/main/resources/assets/schemas/Dac.yaml`